### DISCUSSION TITLE
Where can I find iCubGenova02 motor models and datasheets?

### DISCUSSION THREAD

**Discussion by isorrentino** [robotology/community#535] [Q&A] [answered]
Hi all,

I was looking for the datasheet of the iCubGenova02 motor models and datasheets. Thanks in advance for your help.

cc @traversaro @pattacini @salvi-mattia

**Comment by salvi-mattia** [comment, marked as answer]
these pieces of information are not structured in our doc site at the moment.

as you told me on teams, you need legs' data:

hip roll: MOOG_C2900584
hip yaw: MOOG_C2900584
knee: MOOG_C2900584
ankle pitch: MOOG_C2900584
ankle roll: MOOG_C2900584

lemme know if this answer your question

**Reply by isorrentino** [reply]
The hip pitch motor is the same as the hip roll one? @salvi-mattia

**Reply by isorrentino** [reply]
@salvi-mattia Is there an error in the previous comment? I noticed that the motor models are the same for each joint. Is it correct?

**Reply by salvi-mattia** [reply]
yes the hip pitch is the same of the hip roll....and the same of all the others

**Reply by fiorisi** [reply]
@isorrentino you can find more details about the leg joints [here](https://github.com/robotology/community/discussions/256).

**Reply by isorrentino** [reply]
Thank you @fiorisi
